### ISSUE TITLE
[Feature] "Last updated column" and "Mark mod option"

### ISSUE THREAD

**Issue by merket** [KSP-CKAN/CKAN#2901]
Suggestions
-----------
**Describe the solution you'd like**
Feature request 1: Last updated date column
I believe it would be wonderful to have another column for "Last updated date" and to be able to sort by it.

This way we would be able to check at a glance when was it last updated. Because sometimes modders mark their mod compatible with a certain version but it turns out they're not (i.e. firespitter)

Or we could use this column to discover new mods, seeing which ones are truly up to date and maintained or perhaps newly arrived...

Feature request 2: "Mark selected mod" or "Follow mod" for future alerts

Especially after the official updates and when most of the mods getting broken, we may have to remove some mods from our list due to their incompatibilities, but we don't want to forget about it or would like to get and alert when they're updated, _like as if they're were installed_. Rather than manually tracking and checking everyday.

Or sometimes we simply have our eyes laid on a particular mod that we really want to try out, but just don't feel ready yet for all of their complications, settings, or learning curves etc. So we may just simply want to mark it for later consideration.

The marking option can simply be a line highlight, like green or yellow or custom....

**Additional context**
I believe these requested two features would add quality to every user's life.  Despite not being a coder myself, I can almost imagine them not being impossibly hard to implement.

It would be wonderful to be seen and heard about these here, I also would LOVE to see them added to CKAN client in the future and am sure that other people would appreciate that update as well.

I thank you for still maintaining this wonderful tool and wish everybody involved with the project, a great day!

**Comment by HebaruSan** [comment]
I believe the oldest duplicate we have for part 1 is #1155; we don't need another issue to track that, so we may prune that from this issue. Some work has been done on this in KSP-CKAN/NetKAN-Infra#42, but it's tied up in debates over design choices.

Are you aware of the "Newly compatible" filter? This shows which mods have become compatible with your game version since the last time you refreshed, including both newly indexed mods and updated mods. Useful during disruptive updates like KSP 1.8.0:

![image](https://user-images.githubusercontent.com/1559108/67349638-2031e980-f50e-11e9-8ed5-2d6020c04553.png)
![image](https://user-images.githubusercontent.com/1559108/67349605-0395b180-f50e-11e9-9798-347a661ba483.png)

**Comment by merket** [comment]
> I believe the oldest duplicate we have for part 1 is #1155; we don't need another issue to track that, 

Glad to see this was mentioned way back in 2015.
I always wanted this feature from the first day I started using CKAN, but only recently decided to suggest it. Sorry it was a duplicate.

> Some work has been done on this in [KSP-CKAN/NetKAN-Infra#42](https://github.com/KSP-CKAN/NetKAN-Infra/pull/42), but it's tied up in debates over design choices.

 I don't understand what "the bot being stateless" means but it's great to know that some steps are taken towards implementing this feature.

Since there's a "Latest version" column in the client which always updates dynamically, I always thought it would be as simple as dynamically importing the  `<relative-time datetime>` table from GitHub releases page. Of course not every mod uses GitHub but you get the idea...

Turns out it's much deeper than that and I'll never truly understand how, since I'm not a coder.

> Are you aware of the "Newly compatible" filter? This shows which mods have become compatible with your game version since the last time you refreshed, including both newly indexed mods and updated mods. Useful during disruptive updates like KSP 1.8.0:

Yes, I was already aware of that filter, but it's always at 0 for me. Learning now that it's "since the last refresh" I'm thinking having the auto-refresh setting, resetting it in the background every time. So I was never able to benefit from this filter, but I will pay more attention to it from now on.

As for my second feature wish, I hope it's much easier to implement than my first one.

Thank you so much for your reply.
Have a nice day.
